### ISSUE TITLE
Menu Navigation: Screen Reader jumps unavailable items

### ISSUE THREAD

**Issue by jmwmusic** [musescore/MuseScore#32413]
### Issue type

Accessibility issue (e.g. for keyboard-only or screen reader users)

### Description with steps to reproduce

If an item in the menus is unavailable the screen reader is jumping straight over it.  This makes it impossible for a blind user to browse through all options unless they know they have a score open that every option is possible for.  It is usual for these items to be navigated to but the disabled state announced by the screen reader, e.g. NVDA says "Not Available".

Steps:
1.  Open MuseScore 4 but do not open a score.
2. Press Alt T to open the file menu.
3. Arrow down several times and you will see that navigation jumps between voices menu and Bars menu.  All other dimmed items are skipped.

### Supporting files, videos and screenshots

N/A

### In which versions of MuseScore Studio is this issue present?

4.6

### Regression

I was unable to check

### Operating system

Windows 11 with NVDA

### Checklist

- [x] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [x] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [x] I have attached all requested files and information to this report
- [x] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible
